content/en/docs/collector/transforming-telemetry.md

```diff
@@ -169,6 +169,39 @@ processors:
   k8sattributes/default:
 ```
 
+## Set status for span
+
+**Processor**:
+[span processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)
+
+The
+[span processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)
+can be used to change span status.
+
+The following setting is required:
+
+- `code:` Represents span status. One of the following values `Unset`, `Error`,
+  `Ok`.
+- `description:` setting is allowed only for code `Error`.
+
+Example:
+
+```yaml
```

Comment:
I would prefer a more realistic example like this:

```
span/set_status_ok:
  include:
    attributes:
      - key: http.status_code
        value: 400
  status:
    code: "Ok"
```

The example you have below unconditionally sets all spans to an error which is not a scenario most people will look for. They will instead likely look for either a specific span or a condition in which to change the status.